content/blog/2017/2017-02-14-pipleine-editor-preview.adoc

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "Say 'Hello' to the Blue Ocean Pipeline Editor"
+tags:
+- blueocean
+- editor
+- declarative
+- pipeline
+author: jamesdumay
+---
+
+link:/blog/2016/09/19/blueocean-beta-declarative-pipeline-pipeline-editor/[Back in September 2016 we announced the availability of the Blue Ocean beta]
+and the forthcoming Pipeline Editor. We are happy to announce that you can try
+the Pipeline Editor preview today.
+
+=== What is it?
+
+The Pipeline Editor is the simplest way for anyone wanting to get started with
+creating Pipelines in Jenkins. It's also great way for advanced Jenkins users
+to start adopting Pipeline. It allows developers to break up their Pipeline into different
+ stages and parallelize tasks that can occur at the same time - graphically.
+ The rest is up to you.
+
+A pipeline you create or edit visually can produce a link://blog/2017/02/03/declarative-pipeline-ga/[Declarative Pipeline]Jenkinsfile for you. This is a "round trip" editor which lets you load the pipeline back in later, for more editing. Not sure what a Jenkins Pipeline is? Why not check out our link:/doc/[new guided tour]!
+
+image::/images/blueocean/editor-1.png["The Editor", role="center"]
+
+=== Whats next?
+
+`Jenkinsfile`'s belong in the Git repository so that it is versioned a long side
+your application code. In Blue Ocean, you will be able to edit a `Jenkinsfile`
+for a branch directly from within the user interface using the Editor.
```

Comment:
This sentence uses "edit" whereas the next one uses "author". To me those mean two different things. If I can **author** a Pipeline, that implies I would be able to create `Jenkinsfile` from scratch. Whereas **editing** implies to me that I can edit an existing `Jenkinsfile` loaded from the repository.

Are both correct  insofar that both are functionalities the Visual Pipeline Editor will provide?